ui/src/common/GitHubIcon.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import SvgIcon from '@material-ui/core/SvgIcon';
+
+export default function GitHubIcon() {
```

Comment:
If this Icon is available in a newer version feel free to update it :D.

Reply:
Ok, thanks :+1: